cmd/ci-operator-prowgen/main.go

```diff
@@ -56,8 +56,9 @@ exec ci-operator \
     --template=/tmp/%[1]s
 `
 
-	sshKeyPath    = "/usr/local/github-ssh-credentials-openshift-bot"
-	sshSecretName = "github-ssh-credentials-openshift-bot"
+	oauthTokenPath  = "/usr/local/github-oauth-credentials-openshift-bot"
```

Comment:
This still mentions `openshift-bot`, I think this can easily be called just `github-oauth-credentials`